README.md

```diff
@@ -279,6 +279,21 @@ example:
 This option should be used carefully since disabling the verification of the
 remote system's certificate can subject knife commands to spoofing attacks.
 
+##### Connecting securely to self-signed certs
+If you generate a self-signed cert, the fqdn and ip may not match which will result in a certificate validation failure. In order to securely connect and reduce the risk of a "Man In The Middle" attack, you may use the certificate's fingerprint to precisely identify the known certificate on the WinRM endpoint. You can extract this fingerprint from a listening server via the openssl s_client command:
+
+    openssl s_client -showcerts -connect $IP:5986 < /dev/null 2>/dev/null | \
+						openssl x509 -sha1 -fingerprint -noout | sed -e 's/^.*=//;s/://g'
```

Comment:
Wouldn't this be subject to the same potential MITM challenges unless you run from the server hosting the ssl endpoint?